create/creating-applications-with-cicd-pipelines.adoc

```diff
@@ -27,7 +27,7 @@ This section uses the `pipelines-tutorial` example to demonstrate the preceding
 == Prerequisites
```

Comment:
@mramendi: Refer to https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#anchoring-prerequisites-additional-resources-and-next-steps-titles-in-assemblies

Reply:
Again not in scope as this is pre-existing content. Changing this would require a full review cycle